File: Selenium/Activities/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/alerts")
    title=driver.title
    print("Title of the page is: ",title)

    wait = WebDriverWait(driver, timeout=10)

    driver.find_element(By.ID,"prompt").click()

    alert=wait.until(EC.alert_is_present())

    print("Text in Alert is: "+alert.text)

    alert.send_keys("Awesome!")
    alert.accept()

    print("Message is: ",driver.find_element(By.ID,"result").text)

File: Selenium/Activities/Activity20.java
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
public class Activity20 {

	public static void main(String[] args) {
		   WebDriver driver=new FirefoxDriver();
		
	       driver.get("https://training-support.net/webelements/alerts");
			
		   String title=driver.getTitle();
		   System.out.println("Title of the page is: "+title);
		   
           driver.findElement(By.id("prompt")).click();
		   
		   Alert alert=driver.switchTo().alert();
		    
		   String alertText=alert.getText();
		   System.out.println("Text in Alert is: "+alertText);
		   
		   alert.sendKeys("Awesome!");
		   
		   alert.accept();
		   
		   String message=driver.findElement(By.id("result")).getText();
		   System.out.println("Message after accepting alert is: "+message);
		   
		   driver.quit();

	}

}
